tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/unit_normalization_test.py

```diff
@@ -0,0 +1,107 @@
+# Python imports
+import unittest
+
+from cdr_cleaner.cleaning_rules.unit_normalization import UnitNormalization, \
+    UNIT_NORMALIZATION_QUERY, SANDBOX_UNITS_QUERY, UNIT_MAPPING_TABLE, MEASUREMENT
+# Project imports
+from constants.bq_utils import WRITE_TRUNCATE
+from constants.cdr_cleaner import clean_cdr as clean_consts
+
+QUERY = 'query'
+
+
+class UnitNormalizationTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = 'foo_project'
+        self.dataset_id = 'foo_dataset'
+        self.sandbox_id = 'foo_sandbox'
+        self.client = None
```

Comment:
I think this is not used and can be removed.

Reply:
These are being used in setup_rule tests.